pkg/kubelet/volume_manager.go

```diff
@@ -17,6 +17,7 @@ limitations under the License.
 package kubelet
 
 import (
+	"k8s.io/kubernetes/pkg/api"
```

Comment:
nit: import order is Go packages first (like 'sync'), line break, then kube packages.  generally all by alpha.